web/thesauruses/meta_info.json

```diff
@@ -5,6 +5,7 @@
     "c": "C",
```

Comment:
And same for these changes.

Reply:
@geekygirlsarah Sure thing, I will just do necessary change which is in view.py asap, thank you !